src/Ocelot/Requester/HttpClientBuilder.cs

```diff
@@ -42,15 +42,29 @@ public IHttpClient Create(DownstreamContext context)
             {
                 return httpClient;
             }
-
-            var httpclientHandler = new HttpClientHandler
+            bool useCookies = context.DownstreamReRoute.HttpHandlerOptions.UseCookieContainer;
+            HttpClientHandler httpclientHandler;
+            // Dont' create the CookieContainer if UseCookies is not set ot the HttpClient will complain
```

Comment:
should this be in a preprocessor directive if the problem only exists in .net 461? You can read more about them [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) might make more sense rather than the comment